linux-tkg-config/prepare

```diff
@@ -1029,7 +1034,7 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\r
 
   # Default to 500Hz if _timer_freq is not set
   if [ -z "$_timer_freq" ]; then
-    _default_index="${_default_timer_frequencies_index[$_cpusched]}"
+    _default_index="${_default_timer_frequencies_index[${_cpusched:-cfs}]}"
```

Comment:
This default needed to be set as default when unset due to `_vanilla_build` skipping `_set_cpu_scheduler()`